Source/WebCore/loader/ResourceLoadInfo.h

```diff
@@ -35,12 +35,12 @@ namespace WebCore::ContentExtensions {
 
 enum class ActionCondition : uint32_t {
     None = 0x00000,
-    IfTopURL = 0x20000,
-    UnlessTopURL = 0x40000,
-    IfFrameURL = 0x60000,
-    UnlessFrameURL = 0x80000
+    IfTopURL = 0x80000,
+    UnlessTopURL = 0x100000,
+    IfFrameURL = 0x180000,
+    UnlessFrameURL = 0x200000
 };
-static constexpr uint32_t ActionConditionMask = 0xE0000;
+static constexpr uint32_t ActionConditionMask = 0x380000;
 
 enum class ResourceType : uint32_t {
     Document = 0x0001,
```

Comment:
If we are splitting this into TopDocument and ChildDocument, we should remove this and only use 1 additional bit instead of 2 additional bits.

Also, I think the addition of top-document and child-document should be split into its own PR.

Reply:
Good point, then existing content blockers that still use `"document"` can parse into `TopDocument | ChildDocument`.